src/Core/Addresses/Models/Address.php

```diff
@@ -49,4 +50,9 @@ public function user()
     {
         return $this->belongsTo(User::class);
     }
+
+    public function type(): string
```

Comment:
Ah, small thing, I know it's not wrong but we haven't used return types

Reply:
Ah, will get rid of it. Two secs